hardware_interface/test/test_component_interfaces.cpp

```diff
@@ -104,6 +104,13 @@ class DummyActuator : public hardware_interface::ActuatorInterface
     return hardware_interface::return_type::OK;
   }
 
+  hardware_interface::return_type accept_command_resource_claim(
+    const std::vector<std::string> & interfaces) override
+  {
+    (void)interfaces;
```

Comment:
```suggestion
  hardware_interface::return_type accept_command_resource_claim(
    const std::vector<std::string> & /*interfaces*/) override
  {
```
It is good practice to avoid hacky code to make the linters happy

Reply:
that is a much neater way of handling it! Thanks, fixed in all relevant files now :)